amf-webapi/shared/src/main/scala/amf/plugins/document/webapi/parser/spec/domain/OasContentParser.scala

```diff
@@ -35,7 +35,7 @@ case class OasContentParser(entry: YMapEntry, producer: Option[String] => Payloa
     val mediaTypeText: String = mediaTypeNode.text().toString
```

Comment:
[Out of scope] a getMediaType method could be here that gets the key and returns a string.